app/service/plugin/PluginManager.php

```diff
@@ -580,12 +530,16 @@ private function hasWildcardPermission(string $slug, string $permission): bool
         // 检查是否有通配符权限
         foreach ($granted as $grantedPerm) {
```

Comment:
改进了通配符权限匹配逻辑，分别处理了:*和.*两种通配符形式，并且更精确地匹配前缀。这增强了权限系统的灵活性和准确性。

```suggestion
// 检查是否有通配符权限
foreach ($granted as $grantedPerm) {
    // 支持 :* 和 .* 两种通配符形式
    if (str_ends_with($grantedPerm, ':*')) {
        $prefix = substr($grantedPerm, 0, -2); // 移除 :*
        // 确保精确匹配前缀，检查完整前缀 + 冒号
        if (str_starts_with($permission, $prefix . ':')) {
            return true;
        }
    } elseif (str_ends_with($grantedPerm, '.*')) {
        $prefix = substr($grantedPerm, 0, -2); // 移除 .*
        // 确保精确匹配前缀，检查完整前缀 + 点
        if (str_starts_with($permission, $prefix . '.')) {
            return true;
        }
    }
}